### PR TITLE
November 17 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | c6f890c |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-369.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20231117_amplify.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+The Embedded AWS Gateway, Apigee X, and GitHub Discovery agents have been released.  There are no enhancements or bug fixes in this release.
+
+#### Bug fixes for SaaS (embedded) agents
+
+* **Security and stability updates** - Security and stability fixes have been included for all SaaS (embedded) agents.
+
+### Service Registry
+
+There are no new enhancements or bug fixes in this release.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231117_amplify.md:99]: remove this line

File: content/en/docs/amplify_relnotes/20231117_amplify.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+There are no known limitations in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Add the stage of a resource when creating/editing/viewing a product plan quota** - To help provider to be aware of what he is adding to a plan quota, the stage information of the API Service is now visible.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231117_amplify.md:130]: The stage of an API resource, that shows where the API resource is deployed, is now visible on the product plan quota screen. This helps producers to determine how they want to charge for each endpoint.

File: content/en/docs/amplify_relnotes/20231117_amplify.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Oauth authentication flow support** - It is now possible to request an OAuth token from the Marketplace Try out screen. This will navigate the user to the authentication screen and back to the Marketplace once the authentication succeed.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231117_amplify.md:152]: replace Oauth with OAuth

**Pull request merged**